serverjs/updatecards.js

```diff
@@ -1,7 +1,9 @@
 var util = require('./util.js');
 var carddb = require('./cards.js');
 const fs = require('fs');
+const path = require('path');
 const https = require('https');
+const cardutil = require('../dist/util/Card.js');
```

Comment:
this line is causing tests to fail, is this directory not visible when testing? is it a casing issue?